plugins/inputs/aerospike/aerospike.go

```diff
@@ -186,6 +242,30 @@ func copyTags(m map[string]string) map[string]string {
 	return out
 }
 
+func splitNamespaceAndOperation(hist string) (string, string) {
+	if strings.Contains(hist, "{") {
+		indexOfOpenCurlyBracket := strings.Index(hist, "{")
+		indexOfCloseCurlyBracket := strings.Index(hist, "}")
+		namespace := hist[indexOfOpenCurlyBracket+1 : indexOfCloseCurlyBracket]
+		operation := hist[indexOfCloseCurlyBracket+2:]
+		return operation, namespace
+	}
+	return hist, "total"
+}
+
+func getDurationOfTransaction(sTimeWithTimeZone string, eTime string) float64 {
+
+	timing := strings.Split(sTimeWithTimeZone, "-")
+	sTime := timing[0]
+	startTime, _ := time.Parse("15:04:05", sTime)
+	endTime, _ := time.Parse("15:04:05", eTime)
+	duration := (endTime.Sub(startTime)).Seconds()
+	if duration < 0 {
+		duration += 86400
```

Comment:
What's the significance of `86400`?

Reply:
24 (hh) * 60 (mm) *60 (ss) = 86400.
eg., get seconds between 23:59:04 and 00:00:04 cases this will help.
You can check in the unit test case.